content/en/integrations/iam_access_analyzer.md

```diff
@@ -0,0 +1,111 @@
+---
```

Comment:
Is this file supposed to be included in this PR? Your comments only mention the new section on the "Set up Tracing on a GitLab Pipeline" page.

Reply:
Hi Ursula, this file is not supposed to be included in this PR - not sure how I managed to add an entire new page of content (might have pulled from somewhere else by accident)